app/controllers/application_controller.rb

```diff
@@ -10,17 +10,34 @@ class ApplicationController < ActionController::Base
   # Smartly redirect user back to URL they requested before authenticating
   # From: https://github.com/heartcombo/devise/wiki/How-To:-Redirect-back-to-current-page-after-sign-in,-sign-out,-sign-up,-update
   before_action :store_user_location!, if: :storable_location?
+  before_action :root_breadcrumb
 
   rescue_from ActionPolicy::Unauthorized, with: :deny_access
 
   authorize :user_with_groups
 
+  attr_reader :breadcrumbs
```

Comment:
Hmm, can you say what this is adding?

Reply:
Isn't this needed so we can access the breadcrumbs array in the application layout (to pass to the component for rendering)?  https://github.com/sul-dlss/happy-heron/pull/885/files#diff-f43fe075643e681b2c01c2f853bb0c4299d135b47fcbd4da96890d521c49e3ebR45

Reply:
I don't think it is. I believe it's accessible by virtue of being an ivar in the controller. You open to commenting it out and verifying?

Reply:
indeed, it works, removing